terraso_backend/config/settings.py

```diff
@@ -236,3 +236,5 @@
 PLAUSIBLE_URL = config("PLAUSIBLE_URL", default="https://plausible.io/api/event")
 RENDER_API_URL = config("RENDER_API_URL", default="https://api.render.com/v1/")
 RENDER_API_TOKEN = config("RENDER_API_TOKEN", default="")
+
+HARDDELETE_DELETION_GAP = config("HARDDELETE_DELETION_GAP_DAYS", default="30", cast=config.eval)
```

Comment:
question: can we pass the date in as `30` instead of `"30"`?

Reply:
Yeah, I think so. I copied what is used elsewhere in the file but I think it would be the same result.